apps/base-docs/tutorials/docs/3_farcaster-frames-gating-and-redirects.md

```diff
@@ -244,42 +202,46 @@ if (message?.button === 3 && message.liked) {
     getFrameHtmlResponse({
       buttons: [
         {
-          label: 'Story time!',
+          label: `State: ${state?.page || 0}`,
         },
         {
           action: 'link',
-          label: 'Link to Google',
-          target: 'https://www.google.com',
+          label: 'OnchainKit',
+          target: 'https://onchainkit.xyz',
         },
         {
-          label: 'Redirect to pictures',
           action: 'post_redirect',
+          label: 'Like to see doggos!',
         },
       ],
       image: {
-        src: `${NEXT_PUBLIC_URL}/park-3.png`,
-        aspectRatio: '1:1',
-      },
-      input: {
-        text: 'Tell me a boat story',
+        src: `${NEXT_PUBLIC_URL}/park-1.png`,
       },
       postUrl: `${NEXT_PUBLIC_URL}/api/frame`,
+      state: {
+        page: state?.page + 1,
+        time: new Date().toISOString(),
+      },
     }),
   );
 }
```

Comment:
I needed to add a default return statement. typescript linter gave me an error

I added:
```
  return new NextResponse('Unhandled request', { status: 400 });
  ```

Reply:
You're supposed to leave the original return of a frame at the bottom.  Added a note.